.eslintrc.cjs

```diff
@@ -1,9 +1,28 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
   root: true,
-  extends: ['@documenso/eslint-config'],
+  extends: [
+    '@documenso/eslint-config',
+    'plugin:package-json/recommended',
```

Comment:
I like this plugin, we should add it to the core eslint-config package 🙌

Reply:
That's the main mistake I made, I was adding it to the `eslint` file file by mistake 😂